.github/workflows/php.yml

```
@@ -0,0 +1,54 @@
+name: CI FOSHttpCacheBundle
```

```suggestion
name: CI
```

This name is used in the build badge in the readme. the simpler it is the better.